packages/govuk-frontend/src/govuk/components/footer/_index.scss

```diff
@@ -73,15 +70,12 @@
 
   .govuk-footer__copyright-logo {
     display: inline-block;
-    min-width: $govuk-footer-crest-image-width;
-    padding-top: ($govuk-footer-crest-image-height + govuk-spacing(2));
+    min-width: $govuk-footer-crest-width;
+    padding-top: ($govuk-footer-crest-height + govuk-spacing(2));
     background-image: govuk-image-url("govuk-crest.png");
-    @include govuk-device-pixel-ratio {
```

Comment:
I've had a reminder from @36degrees that other service teams likely use `govuk-device-pixel-ratio()` so we'll need to stick with our [**Deprecating a Sass function or mixin**](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/managing-change.md#deprecating-a-sass-function-or-mixin) notes to avoid a surprise breaking change

But love that we don't need this now 😊

To help manage that change we can either:

1. Keep the mixin but [add `@deprecated`](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/managing-change.md#deprecating-a-sass-function-or-mixin) with a CHANGELOG entry in this PR
2. Keep the mixin but write up a separate GitHub issue to deprecate it in future

Reply:
Happy to! Given impending v5.0 release, is there an opportunity to deprecate in a prior 4.x release, and remove in v5.0?

If so, I can create a separate issue to remove this mixin and attach it to the 5.0 milestone.

Reply:
We've done a "last" (v4.6) release already and next "last last" (v4.7) we'd like to freeze by tomorrow at 5pm. But if you're up for it we can try and take a quick deprecation-only PR against [`support/4.x`](https://github.com/alphagov/govuk-frontend/tree/support/4.x) branch?

It's a bit of a stretch as time's against us but we massively appreciate the contribution

Alternatively we can deprecate in v5.0 and remove in v5.1 instead?

Reply:
As an example, you can see how we deprecated [`govuk-not-ie8()` on `support/4.x`](https://github.com/alphagov/govuk-frontend/blob/support/4.x/package/govuk/tools/_ie8.scss#L36) but added new private versions too. Means we could continue using them (internally) until ready to remove entirely:

* https://github.com/alphagov/govuk-frontend/pull/3343

Reply:
Challenge accepted! #3877